demo-app/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import java.io.FileInputStream
+import java.util.Properties
+
+plugins {
+    id("otel.android-app-conventions")
+}
+
+val localProperties = Properties()
+localProperties.load(FileInputStream(rootProject.file("local.properties")))
```

Comment:
If I'm not mistaken, I think we don't need to manually load preferences from the `local.preferences` file, I remember it being done automatically in android projects.

Reply:
I think we need this in order to reference those properties in the build script itself (like this does, by putting an access token into the resource values). If we just want to remove that for now we can do too.

Reply:
I see, what I meant is that I think this should work out of the box: `rootProject.property("rum.access.token")` (or maybe `rootProject.extra["rum.access.token"]`, not too sure anymore after the change from groovy to kts 😅) because Android projects _should_ be aware of properties inside a `local.properties` file. However, I haven't checked that functionality in a while so maybe it's not the same anymore... Or maybe I'm mixing things up, anyway I think we can leave it like this too, it's not a blocker. 👍